ime/app/src/main/res/values/settings_keys_dont_translate.xml

```diff
@@ -374,6 +374,8 @@
 
     <string name="settings_key_allow_layouts_to_provide_generic_rows">settings_key_allow_layouts_to_provide_generic_rows</string>
     <string name="settings_key_apply_remote_app_colors">settings_key_apply_remote_app_colors</string>
+    <string name="settings_key_popup_characters_order">settings_key_popup_characters_order</string>
+    <string name="settings_key_popup_characters_order_default">NLOS</string>
```

Comment:
the default value should be in `settings_defaults_dont_translate.xml`

Reply:
It is there. The default value is `NLOS`, line 378.

Is there any other `settings_defaults_dont_translate.xml`?